src/main/java/net/minecraftforge/event/entity/player/PlayerEvent.java

```diff
@@ -180,6 +182,43 @@ public void setDisplayname(ITextComponent displayname)
         }
     }
 
+    /**
+     * TabListNameFormat is fired when a player's display name for the tablist is retrieved.<br>
+     * This event is fired whenever a player's display name for the tablist is retrieved in
+     * {@link ServerPlayerEntity#getTabListDisplayName()} or {@link ServerPlayerEntity#refreshTabListName()}.<br>
+     * <br>
+     * This event is fired via the {@link ForgeEventFactory#getPlayerTabListDisplayName(PlayerEntity)}.<br>
+     * <br>
+     * {@link #displayname} contains the display name of the player or null if the client should determine the display name itself.
+     * <br>
+     * This event is not {@link net.minecraftforge.eventbus.api.Cancelable}.
+     * <br>
+     * This event does not have a result. {@link HasResult}
+     * <br>
+     * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
+     **/
+    public static class TabListNameFormat extends PlayerEvent
+    {
+        @Nullable
+        private ITextComponent displayname;
```

Comment:
Use camel case, e.g. `displayName` (applies to other areas in this class too)

Reply:
I copied that name from the `NameFormat` event but I can change this